packages/@webex/internal-plugin-conversation/src/share-activity.js

```diff
@@ -156,7 +156,7 @@ const ShareActivity = WebexPlugin.extend({
    */
   add(file, options) {
     options = options || {};
-    options.claimedFileType = file.displayName.substring(file.displayName.lastIndexOf('.'));
+    options.claimedFileType = file.name.substring(file.name.lastIndexOf('.'));
```

Comment:
displayName changed to name ?

Reply:
It appears this was changed to `name`, the file, at this stage, is a `blob`. See [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/API/File).

Reply:
`displayName` may have been a legacy browser-support item, but it doesn't appear to exist anymore.